devolutions-gateway/src/recording.rs

```diff
@@ -150,8 +178,8 @@ impl ActiveRecordings {
         self.0.lock().contains(&id)
     }
 
-    /// Returns a copy of the internal HashSet
-    pub fn cloned(&self) -> HashSet<Uuid> {
+    /// Get the list of active recordings without locking the mutex
+    pub fn copy_set(&self) -> HashSet<Uuid> {
         self.0.lock().clone()
     }
```

Comment:
It seems you overrode code already present. Revert these changes.

Reply:
updated